lib/goodreads_parser.py

```diff
@@ -26,8 +27,6 @@ def __init__(self, url, connection, asin, raise_error_on_page_not_found=False, c
         self._characters = {}
         self._settings = {}
         self._quotes = []
-        self._author_profile = None
```

Comment:
These were just duplicates - they are initialised later in the file.  [if you haven't seen it, see the commit comments for #73, rather than the pull request - full details there of what I did]